payments/models.py

```diff
@@ -154,3 +169,348 @@ def __str__(self):
             f"{self.service} - {self.period} "
             f"({self.price_value}{'€' if self.price_unit == PriceUnits.AMOUNT else '%'})"
         )
+
+
+class OrderManager(models.Manager):
+    def berth_orders(self):
+        return self.get_queryset().filter(
+            Q(
+                Q(_product_content_type__model=BerthProduct._meta.model_name)
+                | Q(_lease_content_type__model=BerthLease._meta.model_name)
+            )
+        )
+
+    def winter_storage_orders(self):
+        return self.get_queryset().filter(
+            Q(
+                Q(_product_content_type__model=WinterStorageProduct._meta.model_name)
+                | Q(_lease_content_type__model=WinterStorageLease._meta.model_name)
+            )
+        )
+
+
+class Order(UUIDModel, TimeStampedModel):
+    customer = models.ForeignKey(
+        "customers.CustomerProfile",
+        verbose_name=_("customer"),
+        on_delete=models.CASCADE,
+        related_name="orders",
+    )
+    product = GenericForeignKey("_product_content_type", "_product_object_id")
+    lease = GenericForeignKey("_lease_content_type", "_lease_object_id")
+    status = EnumField(OrderStatus, default=OrderStatus.WAITING)
+    comment = models.TextField(blank=True, null=True)
+    price = models.DecimalField(
+        verbose_name=_("price"),
+        max_digits=10,
+        decimal_places=2,
+        validators=[MinValueValidator(Decimal("0.01"))],
+        blank=True,
+    )
+    tax_percentage = models.DecimalField(
+        verbose_name=_("tax percentage"), max_digits=5, decimal_places=2, blank=True,
```

Comment:
should we have validation here? `validators=[MinValueValidator(Decimal("0.00"))],` or something?